core/templates/xregistry_openapi_template.json

```diff
@@ -1,1068 +1,2042 @@
 {
-  "openapi": "3.0.3",
-  "info": {
-    "title": "xRegistry API",
-    "description": "xRegistry API",
-    "version": "0.5-wip"
-  },
-  "servers": [],
-  "security": [
-    {
-      "api_key": []
-    }
-  ],
-  "paths": {
-    "/": {
-      "get": {
-        "operationId": "getRootDocument",
-        "description": "Gets the root document",
-        "parameters": [
-          {
-            "in": "query",
-            "name": "inline",
-            "description": "Set if references shall be inlined",
-            "required": false,
-            "allowEmptyValue": true,
-            "schema": {
-              "type": "boolean"
-            }
-          },
-          {
-            "in": "query",
-            "name": "filter",
-            "description": "Filter criterion(s)",
-            "explode": true,
-            "required": false,
-            "schema": {
-              "type": "array",
-              "items": {
-                "type": "string"
-              }
-            }
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "The root document",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "{%-documentTypeReference-%}"
-                }
-              }
-            }
-          }
+    "openapi": "3.0.3",
+    "info": {
+        "title": "xRegistry API",
+        "description": "xRegistry API. See spec sections (e.g. 'Registry Capabilities', 'Retrieving the Registry Model', 'Doc', etc.) for details.",
+        "version": "0.5-wip"
+    },
+    "servers": [],
+    "security": [
+        {
+            "api_key": []
         }
-      },
-      "post": {
-        "operationId": "upsertDocument",
-        "description": "Uploads a registry document and upserts its contents into the registry",
-        "requestBody": {
-          "description": "A request to create or update the discovery endpoint's collection of endpoints with the given endpoints",
-          "content": {
-            "application/json": {
-              "schema": {
-                "$ref": "{%-documentTypeReference-%}"
-              }
+    ],
+    "paths": {
+        "/": {
+            "get": {
+                "operationId": "getRootDocument",
+                "description": "Gets the root registry document. Query parameters 'inline', 'filter', 'doc', and 'specversion' modify the view. (See spec: Retrieving the Registry, Inline, Filter, and Doc)",
```

Comment:
Minor suggestion: Adding tags to the OpenAPI documents makes the rendered documentation more structured, readable. We can add the Tags also dynamically for each group.

Screenshot how it could look like:

![image](https://github.com/user-attachments/assets/319633e6-b2d7-450b-9017-2c283c62236f)


Added like this:

```suggestion
                "description": "Gets the root registry document. Query parameters 'inline', 'filter', 'doc', and 'specversion' modify the view. (See spec: Retrieving the Registry, Inline, Filter, and Doc)",
                "tags": ["Root"]
```